### ISSUE TITLE
ssr: false useAuth issues

### ISSUE THREAD

**Issue by namdevel** [Hebilicious/authjs-nuxt#36]
the api url returning 404 while use authjs-nuxt with [vite-pwa](https://vite-pwa-org.netlify.app/frameworks/nuxt.html)

**Comment by Hebilicious** [comment]
Hi, please provide a reproduction. I assume that the auth routes needs to be excluded from your service worker.

**Comment by namdevel** [comment]
Maybe need excluded from sevice worker

i need ask another problem, how to access session without middleware, because if i disable middleware the status still unauthenticated

**Comment by Hebilicious** [comment]
@namdevel I need a reproduction to help you.

**Comment by namdevel** [comment]
```
const { signIn, signOut, session, status, cookies } = useAuth()
definePageMeta({ auth: true })

console.log(status.value) // after login = authenticated
```


```
const { signIn, signOut, session, status, cookies } = useAuth()

console.log(status.value) // after login = still unauthenticated
```


i need display user profile as guest before login, and userName discord after login

**Comment by namdevel** [comment]
why if not use `definePageMeta({ auth: true })` the status always unauthenticated ?
i need guest still can acces page with message Hello guest
after login Hello Username

but if not use `definePageMeta({ auth: true })` , the status after login still unauthenticated
if use `definePageMeta({ auth: true })` , the page can't accessed because redirect to login page

**Comment by Hebilicious** [comment]
> why if not use `definePageMeta({ auth: true })` the status always unauthenticated ? i need guest still can acces page with message Hello guest after login Hello Username
> 
> but if not use `definePageMeta({ auth: true })` , the status after login still unauthenticated if use `definePageMeta({ auth: true })` , the page can't accessed because redirect to login page

If you look at the playground/credential, you can see `useAuth` working without `definePageMeta`.
I can't reproduce your issue, please share a repository or a stackblitz.

**Comment by namdevel** [comment]
also i get this in chrome console in new version. i need upload repo after this

```js
Uncaught SyntaxError: The requested module '/_nuxt/node_modules/set-cookie-parser/lib/set-cookie.js?v=8382cb98' does not provide an export named 'splitCookiesString' (at utils.mjs?v=8382cb98:3:10)
```

**Comment by namdevel** [comment]
see this repo [nuxt-auth-discord](https://github.com/afkadev/nuxt-auth-discord)

there is 2 page, index and private, index without middleware, and private with middleware

**Comment by Hebilicious** [comment]
> also i get this in chrome console in new version. i need upload repo after this
> 
> ```js
> Uncaught SyntaxError: The requested module '/_nuxt/node_modules/set-cookie-parser/lib/set-cookie.js?v=8382cb98' does not provide an export named 'splitCookiesString' (at utils.mjs?v=8382cb98:3:10)
> ```

Sorry about this, this got fixed in 0.1.9.

Let me investigate your ssr issue.
